gateleen-hook/src/test/java/org/swisspush/gateleen/hook/mocks/FastFailHttpServerResponse.java

```diff
@@ -0,0 +1,255 @@
+package org.swisspush.gateleen.hook.mocks;
```

Comment:
This class is a duplication of the class _org.swisspush.gateleen.core.http.DummyHttpServerResponse_ and can be removed

Reply:
Not exactly. But you're right and this can definitively be improved somewhat :-)